### DISCUSSION TITLE
Can disabled input appear in handleSubmit

### DISCUSSION THREAD

**Discussion by jzrskc** [react-hook-form/react-hook-form#3936] [General] [answered]
Is it possible to submit the value from a disabled input?
I have an input with a checkbox. If the checkbox is enabled, the default value should be submitted:

`<input defaultValue="wes" name="name" ref={register} disabled={checkbox ? true : false} />`

https://codesandbox.io/s/bitter-cloud-ddcmn?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
use `readonly` :)

**Reply by jzrskc** [reply]
I completely forgot about this option, thanks!
For React users you have to write it with camelcase: `readOnly`

**Reply by drydenwilliams** [reply]
Why can't you have a `disabled` within the form on the `handleSubmit`??

**Reply by andrejunges** [reply]
what about for Checkboxes / Radio boxes, any recommendations? @bluebill1049

**Reply by bluebill1049** [reply]
do you have a codesandbox with the problem? @andrejunges
